Navigation Demo Stopped Working on ROS2 Jazzy

### Robot Model

Turtlebot4 Standard

### ROS distro

Jazzy

### Networking Configuration

Simple Discovery

### OS

Ubuntu 24.04

### Built from source or installed?

Installed

### Package version

ros-jazzy-turtlebot4-navigation 
Version: 2.0.1-1noble.20250430.071125

### Type of issue

Navigation (SLAM, Nav2 etc.)

### Expected behaviour

Behaviour described in the navigation tutorial

### Actual behaviour

TB4 doesn't move to 2D goal pose

### Error messages

```bash
[controller_server-1] [ERROR] [1746882382.323839447] [controller_server]: Caught exception in callback for transition 10
[controller_server-1] [ERROR] [1746882382.323941469] [controller_server]: Original error: Considering footprint in collision checking but no robot footprint provided in the costmap.
[controller_server-1] [WARN] [1746882382.323985645] [controller_server]: Error occurred while doing error handling.
[controller_server-1] [FATAL] [1746882382.323999128] [controller_server]: Lifecycle node controller_server does not have error state implemented
[lifecycle_manager-10] [ERROR] [1746882382.327852169] [lifecycle_manager_navigation]: Failed to change state for node: controller_server
[lifecycle_manager-10] [ERROR] [1746882382.328004271] [lifecycle_manager_navigation]: Failed to bring up all requested nodes. Aborting bringup.
```

### To Reproduce

1. ros2 launch turtlebot4_navigation localization.launch.py map:=office.yaml
2. ros2 launch turtlebot4_navigation nav2.launch.py
3. ros2 launch turtlebot4_viz view_navigation.launch.py

### Other notes

This worked out of the box a few weeks ago when I got the robot. Without changing anything it stopped working.

--- a/turtlebot4_navigation/config/nav2.yaml
+++ b/turtlebot4_navigation/config/nav2.yaml
@@ -135,7 +135,15 @@ local_costmap:
       width: 3
       height: 3
       resolution: 0.06
-      robot_radius: 0.175
+      #robot_radius: 0.175
+      footprint: "[[ 0.189,  0.000],
+                  [ 0.134, -0.134],
+                  [ 0.000, -0.189],
+                  [-0.134, -0.134],
+                  [-0.189,  0.000],
+                  [-0.134,  0.134],
+                  [ 0.000,  0.189],
+                  [ 0.134,  0.134]]"
       plugins: ["static_layer", "voxel_layer", "inflation_layer"]
       inflation_layer:
         plugin: "nav2_costmap_2d::InflationLayer"
@@ -174,7 +182,15 @@ global_costmap:
       publish_frequency: 1.0
       global_frame: map
       robot_base_frame: base_link
-      robot_radius: 0.175
+      #robot_radius: 0.175
+      footprint: "[[ 0.189,  0.000],
+                  [ 0.134, -0.134],
+                  [ 0.000, -0.189],
+                  [-0.134, -0.134],
+                  [-0.189,  0.000],
+                  [-0.134,  0.134],
+                  [ 0.000,  0.189],
+                  [ 0.134,  0.134]]"
       resolution: 0.06
       track_unknown_space: true
       plugins: ["static_layer", "obstacle_layer", "inflation_layer"]

